rclcpp/include/rclcpp/time.hpp

```diff
@@ -37,9 +37,9 @@ class Time
       throw std::runtime_error("RCL_ROS_TIME is currently not implemented.");
       ret = false;
     } else if (ClockT == RCL_SYSTEM_TIME) {
-      ret = rcl_system_time_now(&rcl_now);
+      ret = rcutils_system_time_now(&rcl_now);
```

Comment:
Since you wish to keep the function name, the type and name of the rcl_now should be updated to use rcutils or this code changed to use the rcl time source.